### DISCUSSION TITLE
Couldn't apply

### DISCUSSION THREAD

**Discussion by Mani-Chandra65** [community/community#133919] [GitHub Education]
### Hi everyone,

My account shows that the email is verified. I got rejected twice, and now I can't see a change after clicking cotinue after entering college name. Can anyone please help me with this.
![Screenshot from 2024-07-29 16-59-01](https://github.com/user-attachments/assets/b4823ed5-9ccf-4c38-a02d-2654b2966daf)
It stops here.

**Comment by samus-aran** [comment]
Hi @Mani-Chandra65 

I can confirm that you are actually in the Approved state. I suspect you can not continue because you currently already having a processing Application. Are you able to take a screenshot of the "Approved" DR that should be sitting at the top of the application page.

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by Mani-Chandra65** [reply]
I've tried again using phone yeah I got it now. Thankyou for replying tho
